src/containers/vote/index.ts

```diff
@@ -44,27 +46,22 @@ function useVoteContainer(): UseVoteContainer {
   }, [votedStakeAmount, totalVotedStakeAmount]);
 
   const votedPercent = useMemo(() => {
-    const _percent = Number(_votedPercent);
-    const lastProgress = PROGRESS[PROGRESS.length - 1];
-    if (voteFinishedAt) {
-      if (_percent >= lastProgress) return _votedPercent;
-      return lastProgress.toFixed(2);
-    }
     return _votedPercent;
-  }, [_votedPercent, voteFinishedAt]);
+  }, [_votedPercent]);
 
   const getTotalVotedStake = useCallback(async () => {
     const data = await viewFunction({
       contractId: contractId,
       method: 'get_total_voted_stake',
     });
     logger.debug('get_total_voted_stake', data);
-    if (!Array.isArray(data) && data.length !== 2) {
+    if (!Array.isArray(data) || data.length !== 3) {
       logger.error('get_total_voted_stake error', data);
       return;
     }
-    setVotedStakeAmount(Big(data[0]));
-    setTotalVotedStakeAmount(Big(data[1]));
+    setVotedYeaStakeAmount(Big(data[0]));
+    setVotedStakeAmount(Big(data[1]));
+    setTotalVotedStakeAmount(Big(data[2]));
```

Comment:
This is the total stake amount, including what is not voted. I think `totalStakeAmount` will be more appropriate.